cypress/commons/actions/generic/AppBar.js

```diff
@@ -53,7 +61,13 @@ function switchLanguageTo(lang) {
 
 // Help menu actions
 function openHelpMenu() {
-  return getHelpMenu().click();
+  getHelpMenu().click();
+}
+function openAboutDialog() {
+  getAboutButton().click();
+}
+function closeAboutDialog() {
+  getAboutDialogCloseButton().click();
```

Comment:
Could we add the `return` keyword in these functions to be able to chain function calls in the tests files if we need it ?